integration_tests/client/datasets/test_dataset.py

```diff
@@ -272,17 +272,15 @@ def test_create_tabular_dataset_and_add_groundtruth(
     assert set(d.uid for d in data) == {"uid1", "uid2"}
 
     # check metadata is there
-    metadata_links = data[0].meta
-    assert len(metadata_links) == 1
-    assert "metadatum1" in metadata_links
-    assert metadata_links["metadatum1"] == "temporary"
```

Comment:
can you add back this assertion to make sure this change equivalent to the old test?

Reply:
The value isnt defined in the test so im not sure why this is necessary. The assertion is handled by `datum.meta == md1` or `datum.meta == md2`

Reply:
There is no regression in test coverage so im confused why this matters.

Reply:
makes sense!